managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -240,7 +240,6 @@ protected void recoverFromLedger(final ManagedCursorInfo info, final VoidCallbac
             }
 
             // Read the last entry in the ledger
-            cursorLedger = lh;
```

Comment:
so, if we don't initialize `cursorLedger` at the time recovery then it will be null initially and at that time [internal-stat](https://github.com/yahoo/pulsar/blob/master/pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/persistent/PersistentTopic.java#L1052) will not return correct value of `cursorLedgerLastEntry`. So, is there any specific reason to not initialize it at recovery time?